pkgs/by-name/ba/bat/package.nix

```diff
@@ -12,15 +11,15 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "bat";
-  version = "0.24.0";
+  version = "0.25.0";
 
   src = fetchFromGitHub {
     owner = "sharkdp";
     repo = "bat";
     rev = "v${version}";
-    hash = "sha256-1RjlJEmY/jMf0IYQbrWrT1CHFyiqgarOl72u9xjjQiQ=";
+    hash = "sha256-82IhLhw0TdaMh21phBxcUZ5JI5xOXb0DrwnBmPwyfAQ=";
   };
```

Comment:
```suggestion
  };

```

Reply:
I think I took care of this, but github isn't detecting the diff correctly